src/Entity/User/AdminUser.php

```diff
@@ -21,22 +21,26 @@
 class AdminUser extends BaseUser implements AdminUserInterface
 {
     /**
-     * @var string
+     * @var string|null
      *
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $lastName;
+    private $lastName;
```

Comment:
I think you should keep these fields open

Reply:
Why? Admin user doesn't need to be override on the starter.